pkg/controller/daemon/daemoncontroller.go

```diff
@@ -580,6 +582,12 @@ func (dsc *DaemonSetsController) updateDaemonSetStatus(ds *extensions.DaemonSet)
 			desiredNumberScheduled++
 			if scheduled {
 				currentNumberScheduled++
+				// Sort the daemon pods by creation time, so the the oldest is first.
+				daemonPods, _ := nodeToDaemonPods[node.Name]
+				sort.Sort(podByCreationTimestamp(daemonPods))
```

Comment:
I'm using the same approach as here https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/daemon/daemoncontroller.go#L458-L460
If `dsc.getNodesToDaemonPods(ds)` returns more than one pod on node I want to get status of the youngest pod.